server/views/pages/githubRepoRequestForm.njk

```diff
@@ -29,6 +29,14 @@
         text: "Github Repository Name (Max 100 characters, only allowed special character Hyphen(-))"
       }
     }) }}
+    {% if validationResults.github_repo and validationResults.github_repo %}
```

Comment:
there's a standard pattern for showing errors: [link](https://design-system.service.gov.uk/patterns/validation/)

which uses [error summary component](https://design-system.service.gov.uk/components/error-summary/)  to show messages at the top of the page

and reports [error messages](https://design-system.service.gov.uk/components/error-message/) on inputs

For the event summary you probably want to [iterate over the map](https://mozilla.github.io/nunjucks/templating.html#for:~:text=%25%20for%20fruit%2C%20color%20in%20fruits%20%25%7D)

Reply:
Sorted